doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-rest-api-con.adoc

```diff
@@ -0,0 +1,108 @@
+[id='controller-rest-api-con_{context}']
+= {CONTROLLER} REST API for {KIE_SERVER} templates and instances
```

Comment:
I've indeed noted it in the docs, in the intro paragraph to this section. Unfortunately because the API continues to use "templates", "instances", and "containers", we have to continue to refer to those as the primary terms with "configurarions", "remote servers", and "deployment units" as merely parenthetical mentions. Otherwise it will be confusing when people use the API. Ideally, at some point we should update the API so that it uses the same terms as the UI, but I'm told by @mswiderski that that will create backward incompatibility issues.